app/mailers/user_mailer.rb

```diff
@@ -11,6 +11,6 @@ def reset_password_email(user)
 
   def share_link_email(share_link)
     @share_link = share_link
-    mail(:to => share_link.user.email, :reply_to => share_link.user.email, :bcc => share_link.emails, :subject => t(:share_link_email_subject, :email => share_link.user.email))
+    mail(:to => share_link.user.email, :reply_to => share_link.user.email, :bcc => share_link.emails, :subject => t(:share_link_email_subject, :email => share_link.user.email+"("+share_link.user.name+")"))
```

Comment:
I'd prefer it if you do it like this:

    :email => "#{share_link.user.email} (#{share_link.user.name})"